src/api/Config/types.ts

```diff
@@ -105,10 +106,18 @@ export interface IAuditTrailConfig {
   logPlugins?: boolean;
 }
 
+export interface IUnifiedCaseListPreferencesConfig {
+  alertIdsFilter?: number[];
+  caseStatusesFilter: CaseStatus[];
+  assignedUserIdsFilter?: number[];
+  sortBy: FullCaseListSortBy[];
+}
+
 export interface IUserPreferencesConfig {
   locale: string;
   pinOnDrag: boolean;
   incrementalLayout: boolean;
+  unifiedCaseListPreferences: IUnifiedCaseListPreferencesConfig;
```

Comment:
No need for the re-declared interface `IUnifiedCaseListPreferencesConfig`
We can just use the one from `src/api/User/types`